test/test_redis_lock.py

```diff
@@ -253,7 +253,7 @@ def test_no_redis(self):
             redis_port=None,
         )
         mock_func = MagicMock()
-        resulted = wrap_with_run_lock(func=mock_func, redis_params=redis_params)(123, b='abc')
+        resulted = wrap_with_load_lock(func=mock_func, redis_params=redis_params)(123, b='abc')
```

Comment:
TestWrapWithLoadLock was suppose to test `wrap_with_load_lock()`, but accidentally tested `wrap_with_run_lock()` in mistake.